airflow/providers/amazon/aws/hooks/sagemaker.py

```diff
@@ -954,3 +954,13 @@ def _list_request(
                 return results
             else:
                 next_token = response["NextToken"]
+
+    def find_processing_job_by_name(self, processing_job_name: str) -> bool:
+        """
+        Query processing job existence by describe_processing_job, rather than iterate all the jobs.
+        """
```

Comment:
```suggestion
        """Query processing job existence."""
```

No need to explain something wrong if the solution is obviously correct.